index.html

```diff
@@ -63,8 +63,447 @@ <h2 class="who-am-i">I’m a software developer</h2>
+                        </button>
+                       
+                    </li>
+                </ul>
+
+                
+
+                <button class="button project-button" type="button">
+                    See Project
+                </button>
+
+            </div>
+        </section>
+
         
+        <section class="place-holder page-2">
+            
+            <div class="post-stories">
+                <h2 class="multi-post">
+                    Multi-Post Stories 
+                </h2>
+                <div class="gain">
+                    Gain+Glory
+                </div>
+
+                <ul class="lang">
+                    <li class="ruby">
+                        <button class="button" type="button">
+                            Ruby on rails
+                        </button>
+                        
+                    </li>
+    
+                    <li class="css">
+                        <button class="button" type="button">
+                            css
+                        </button>
+                        
+                    </li>
+    
+                    <li class="java">
+                        <button class="button" type="button">
+                            JavScript
+                        </button>
+                        
+                    </li>
+    
+                    <li class="html">
+                        <button class="button" type="button">
+                            html
+                        </button>
+                       
+                    </li>
+                </ul>
+
+                
+
+                <button class="project-button button" type="button">
+                    See Project
+                </button>
+
+            </div>
+        </section>
+
+        <section class="place-holder page-3">
+            
+            <div class="post-stories">
+                <h2 class="multi-post">
+                    Multi-Post Stories
+                </h2>
+                <div class="gain">
+                    Gain+Glory
+                </div>
+                <ul class="lang">
+                    <li class="ruby">
+                        <button class="button" type="button">
+                            Ruby on rails
+                        </button>
+                        
+                    </li>
+    
+                    <li class="css">
+                        <button class="button" type="button">
+                            css
+                        </button>
+                        
+                    </li>
+    
+                    <li class="java">
+                        <button class="button" type="button">
+                            JavScript
+                        </button>
+                        
+                    </li>
+    
+                    <li class="html">
+                        <button class="button" type="button">
+                            html
+                        </button>
```

Comment:
- [ ] Here as well. I would recommend removing the `button` tag here. Since this is just a text and not some button that will work anything.

Reply:
Button tag has been removed.